generate_benchmark.py

```diff
@@ -1,19 +1,37 @@
 #!/usr/bin/env python
 
 import os, sys
+from subprocess import Popen
 import shutil
 
+script_name = "script.py"
+benchmark_descr_dir = "future" # directory with description of all benchmarks
+benchmark_file = "benchmark"
 def main():
     new_dir_name = sys.argv[1]
-    shutil.copytree('future', new_dir_name)
+    shutil.copytree(benchmark_descr_dir, new_dir_name)
     os.chdir(new_dir_name)
-    dir_names = ["pr2-in-iai-kitchen","hrp2-on-the-ground",  "baxter-manipulation-boxes", "construction-set", "pr2-manipulation-kitchen", "pr2-manipulation-two-hand",
-    "romeo-placard", "ur3-spheres"]
+    # note that "pr2-in-iai-kitchen" and "pyrene-on-the-ground" do not use
+    # the manipulation plugin ("manipulation-corba.so") so we should either
+    # restart hppcorbaserver each time, or to run these two benchmarks before
+    # running the rest which loads the plugin
+
+    # get names of all immediate subdirectories that contain a script.py file
+    dir_names = [f.name for f in os.scandir(".") if f.is_dir()
+                and os.path.isfile(os.path.join(f.path, script_name))]
```

Comment:
We'll want to switch from `os.path` to `pathlib.Path` at some point, but that's OK for now